src/azimuth/view/paused.c

```diff
@@ -743,15 +743,22 @@ static void draw_upgrades(const az_paused_state_t *state) {
          "is automatically refilled over time by your reactor." :
          state->hovered_upgrade == AZ_UPG_ROCKET_AMMO_00 ?
          (az_upgrades_have_any_bombs(&player->upgrades) ?
+          az_show_extra_weapon_key(state->prefs, 9) ?
+          "Press [9] or [$9] to select rockets, then \n"
+          "hold down [$o] and press [$f] to fire." :
           "Press [9] to select rockets, then hold down [$o] and\n"
           "press [$f] to fire." :
           "Hold down [$o] and press [$f] to fire.") :
          state->hovered_upgrade == AZ_UPG_BOMB_AMMO_00 ?
          (az_upgrades_have_any_rockets(&player->upgrades) ?
+          az_show_extra_weapon_key(state->prefs, 0) ?
+          "Press [0] or [$0] to select bombs,\n"
```

Comment:
I noticed that the show-extra-weapon-key description for rockets has the "then" on the first line, while the description here for bombs has it on the second line.  Could you move the "then" up to the first line here for consistency?

Reply:
done.  also removed a trailing space on the rockets line.